python/hail/representation/variant.py

```diff
@@ -145,15 +145,17 @@ def num_alleles(self):
     def allele(self, i):
         """Returns the string allele representation for the ith allele.
 
-         The reference is included in the allele index.  The index of
-         the first alternate allele is 1.  The following is true for all
-         variants:
+        The reference is included in the allele index.  The index of
+        the first alternate allele is 1.  The following is true for all
+        variants:
 
-         >>> v.ref == v.allele(0)
+        >>> v = Variant.parse("16:2478434:A:T,TA")
```

Comment:
I think this point no longer makes as much sense when you have to construct a variant here.  Maybe move the variant constructors above?